chocolatey/plugins/modules/win_chocolatey_facts.py

```diff
@@ -133,9 +183,16 @@
               returned: always
               type: str
               sample: username
+        gather_facts:
+          description: The list of subsets that were gathered
+          returned: always
+          type: list
+          elements: str
+          sample: ['all']
+          version_added: '1.5.0'
```

Comment:
This doesn't match the name of the property being returned from the module.

Reply:
Good catch, I don't know what I must have been looking at when I typed that one